administration/src/mui-modules/base/BaseMenu.tsx

```diff
@@ -40,9 +40,14 @@ const BaseMenu = (props: BaseMenuProps): ReactElement => {
         variant='contained'
         onClick={handleMenuOpen}
         color='default'
-        endIcon={open ? <KeyboardArrowUp /> : <KeyboardArrowDown />}
-        sx={{ width: props.containerWidth, height: props.itemHeight }}>
+        sx={{
+          width: props.containerWidth,
+          height: props.itemHeight,
+          justifyContent: 'space-between',
+          paddingRight: 1,
+        }}>
         <Typography variant='button'>{props.menuLabel}</Typography>
+        {open ? <KeyboardArrowUp /> : <KeyboardArrowDown />}
```

Comment:
❓ Is there a specific reason to add the icon as a child elements vs the `endIcon` property? I think the latter is the correct place, it works for RTL languages and is generally the way MUI adds icons to buttons.